OsmAnd/src/net/osmand/plus/OsmandSettings.java

```diff
@@ -1155,6 +1155,8 @@ protected Integer getDefaultValue() {
 
 	// this value string is synchronized with settings_pref.xml preference name
 	public final OsmandPreference<Boolean> SHOW_OSM_BUGS = new BooleanPreference("show_osm_bugs", false).makeGlobal();
+	public final CommonPreference<Boolean> SHOW_CLOSED_OSM_BUGS = new BooleanPreference("show_closed_osm_bugs", false).makeGlobal();
+	public final CommonPreference<Integer> SHOW_OSM_BUGS_MIN_ZOOM = new IntPreference("show_osm_bugs_min_zoom", 8).makeGlobal();
```

Comment:
Settings code here without matching string.xml?

Reply:
These settings should not match anything. I'll add an empty line to separate them.

Reply:
I thought the first parameter is used to find the appropriate localization (and therefor has to match name in string.xml).
But ok. I am no android expert